### PR TITLE
Update associacao-brasileira-de-normas-tecnicas-ufrgs-initials.csl

### PR DESCRIPTION
**Pull request by PriscilaSJUFRGS**
Atualização do estilo associacao-brasileira-de-normas-tecnicas-ufrgs-initials.csl de acordo com a nova norma ABNT 6023:2018

### CODE REVIEW

**Comment by adam3smith** [comment]
Don't worry about the automated test for now -- see my questions and comments on the other pull request and once those are sorted we'll help you fix the automated tests

**Comment by adam3smith** [comment]
@PriscilaSJUFRGS -- I made most of the changes discussed on #4460 here, too, but since you went through several iterations with that style, would be good for you to review this one here again. 
Please note: *If you make changes to this style in this pull request, please start with the latest version from github (under Files Changed)

**Comment by PriscilaSJUFRGS** [comment]
> @PriscilaSJUFRGS -- I made most of the changes discussed on #4460 here, too, but since you went through several iterations with that style, would be good for you to review this one here again.
> Please note: *If you make changes to this style in this pull request, please start with the latest version from github (under Files Changed)

We usually just use the same style of #4460 and change just were it should be abreviate. Can we do that this time too? But only when the other is completed.

**Comment by adam3smith** [comment]
OK, the other style is merged -- please go ahead and make the needed changes to this one, making sure to start from the merged version of the style in #4460 .

File: associacao-brasileira-de-normas-tecnicas-ufrgs-initials.csl [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="pt-BR">
   <info>
-    <title>Universidade Federal do Rio Grande do Sul - ABNT (autoria abreviada) (Portuguese - Brazil)</title>
-    <title-short>UFRGS-ABNT</title-short>
+    <title>Universidade Federal do Rio Grande do Sul - SBUFRGS - ABNT (autoria abreviada) (Portuguese - Brazil)</title>
```
Comment (adam3smith) [inline associacao-brasileira-de-normas-tecnicas-ufrgs-initials.csl]: What's the SB for? that's not in your other styles, and it looks like UFRGS is the standard acronym for the university. Rest looks good

Comment (PriscilaSJUFRGS) [reply associacao-brasileira-de-normas-tecnicas-ufrgs-initials.csl]: It's for "Sistema de Bibliotecas", but UFRGS is better. I atualized in both styles (#4521 and here).
Thanks!

File: associacao-brasileira-de-normas-tecnicas-ufrgs-initials.csl [hunk 2]
```diff
@@ -91,41 +99,66 @@
...
+    <text value="Compositor: "/>
+    <names variable="composer" delimiter="; ">
+      <name delimiter="; " sort-separator=" " delimiter-precedes-last="always"></name>
+      <et-al font-style="italic"/>
+    </names>
+  </macro>
+  <macro name="recipient">
+    <text value="Destinatário: "/>
+    <names variable="recipient" delimiter="; ">
+      <name delimiter=" e " sort-separator=" " delimiter-precedes-last="always"></name>
```
Comment (adam3smith) [inline associacao-brasileira-de-normas-tecnicas-ufrgs-initials.csl]: what's the goal of using "e" here? Wouldn't `and="text"` be better? With " e" as you have it, you'd e.g. get
> Leticia Strehl e Luísia Alves e Michelangelo Viana

whereas I think you probably want

> Leticia Strehl, Luísia Alves e Michelangelo Viana

It is probably rare for letters to have more than two recipients, but might as well get this right. If I'm right and you want the second option above then `<name and="text"/>` would be all you need. It's a bit odd that all the names have different formats, though? E.g. for composers you have 

> Leticia Strehl; Luísia Alves; Michelangelo Viana

Also, as you're currently handling them, the labels like "Compositor: " wouldn't work for multiple composers (which would be "Compositores: " I believe?)
That's what labels are for, along the lines of:
```
<names variable="composer" delimiter="; ">
  <label variabel="composer" text-case="capitalize-first" suffix=": "/>
  <name delimiter="; " sort-separator=" " delimiter-precedes-last="always"></name>
  <et-al font-style="italic"/>
</names>
```

Comment (PriscilaSJUFRGS) [reply associacao-brasileira-de-normas-tecnicas-ufrgs-initials.csl]: You are right! Thanks! But we will have to make these corrections at the next update (estimated for next July).

Comment (PriscilaSJUFRGS) [reply associacao-brasileira-de-normas-tecnicas-ufrgs-initials.csl]: @adam3smith can we merge this style for now?

**Comment by rmzelle** [comment]
Thanks! Sorry for the wait.

**Pull request merged**